Code/Engine/Foundation/Algorithm/Implementation/HashingMurmur_inl.h

```diff
@@ -7,18 +7,18 @@ namespace ezInternal
   template <size_t N, size_t Loop>
   struct CompileTimeMurmurHash
   {
-    constexpr EZ_ALWAYS_INLINE ezUInt32 operator()(ezUInt32 hash, const char (&str)[N], size_t i) const
+    constexpr EZ_ALWAYS_INLINE ezUInt32 operator()(ezUInt32 uiHash, const char (&Str)[N], size_t uiI) const
     {
-      return CompileTimeMurmurHash<N, Loop - 4>()(CompileTimeMurmurHash<N, 4>()(hash, str, i), str, i + 4);
+      return CompileTimeMurmurHash<N, Loop - 4>()(CompileTimeMurmurHash<N, 4>()(uiHash, Str, uiI), Str, uiI + 4);
     }
   };
 
   template <size_t N>
   struct CompileTimeMurmurHash<N, 4>
   {
-    static constexpr EZ_ALWAYS_INLINE ezUInt32 helper(ezUInt32 k) { return (k ^ (k >> MURMUR_R)) * MURMUR_M; }
```

Comment:
Didn't you say single letter variables were allowed?

Reply:
Oh right it's up for debate still. I say single letter variables should be allowed and lower case. This happens a lot in math functions. x^n and stuff like that is easier to read.

Reply:
Same as lhs